modules/nf-core/purecn/intervalfile/meta.yml

```
@@ -0,0 +1,38 @@
+name: "purecn_intervalfile"
+description: Generate on and off-target intervals for PureCN from a list of targets
+keywords:
+  - copy number alteration calling
+tools:
+  - "purecn":
+      description: "Copy number calling and SNV classification using targeted short read sequencing"
+      homepage: "https://bioconductor.org/packages/release/bioc/html/PureCN.html"
+      documentation: "https://bioconductor.org/packages/release/bioc/html/PureCN.html"
+      tool_dev_url: "https://github.com/lima1/PureCN"
+      doi: "10.1186/s13029-016-0060-z."
+      licence: "Artistic-2.0"
+
+input:
+  - target_bed:
+      type: file
+      description: BED file of target intervals
+      pattern: "*.bed"
+  - fasta:
+      type: file
+      description: FASTA reference sequence of the genome being used
+      pattern: "*.fasta"
+  - genome:
+      type: string
+      description: Genome used for the BED file (e.g., "hg38", "mm10"...)
+output:
+  - baits_intervals:
+      type: file
+      pattern: "*.txt"
+      description: |
+        Annotated targets optimized for copy number calling
```

```suggestion
  - txt:
      type: file
      pattern: "*.txt"
      description: |
        Annotated targets optimized for copy number calling
```
Can you also add `bed` with a description of how to generate the file using the tool?